src/components/component-content/component-content.css

```diff
@@ -47,9 +47,14 @@ h3[id^="toc"] + p {
     display: grid;
     grid-template-columns: 1fr 12rem;
     grid-gap: var(--goa-space-2xl);
+
+    @media screen and (max-width: 1230px) {
+        grid-template-columns: 1fr;
+      }
```

Comment:
Does this work? Nesting css is now possible, but last I checked it didn't yet work with the svelte compiler.

If it does work, then just a small fix to the indentation is needed.

Reply:
Yup! It works without issues, and we already use it in the ToC component:

https://github.com/GovAlta/ui-components-docs/blob/988a16991bd86e4a7db9daf13332616353b272d9/src/components/table-of-contents/toc.module.css#L1-L11